README.md

```diff
@@ -1,26 +1,21 @@
 # BuyTogether
+Welcome to our group social project!
+In this project, we have developed a web application that is supposed to help people group up,
+and organize a joint food purchase.
 
-Welcome to our group social project!.
-In this project we have developed a web application which suppose to help people group up
-and organize a large scale food buy.
-Doing so, we "cut" the amount of hands the product go through which will benefit the customer
-with cheaper prices.
-
+Doing so, we remove the middlemen and reduce the price of the food products.
```

Comment:
is this what you meant? because it doesnt look really good, i think the extra line should be deleted.

Reply:
It indeed doesn't look for at the moment. The reason, is the use of the horizontal rule (`---`) afterwards.  I'm not sure exactly what you were originally after when using those.

What happens is that text directly above that `---` line is becoming a heading (becomes bold). If you wanted that entire paragraph in bold, fine. Otherwise, you need to add an extra empty line before the `---`.

See here some details: https://www.markdownguide.org/basic-syntax/#horizontal-rules

The reason I mentioned it originally was that I saw a line break on the text, and I thought you were expecting a line break on the output as well.

Anyhow, it's up to you. Just compare the code and the output and make sure the output is per your expectations.

Reply:
the way it was before is fine by me